highlevel/adapter/socket/tcp.py

```diff
@@ -1,21 +1,21 @@
 """
-Isotp socket can adapter module.
+TCP socket adapter module.
 """
-import asyncio
+
 import binascii
-from asyncio import StreamWriter, StreamReader
+from asyncio import StreamReader, StreamWriter
 from typing import List
 
-from highlevel.logger import LOGGER
 from highlevel.adapter.socket import SocketAdapter, CallbackFunc
+from highlevel.logger import LOGGER
 
 
 class TCPSocketAdapter(SocketAdapter):
     """
     Sends and receive messages from a TCP/IP socket, with the messages being encoded in ASCII:
-    <ABCD>
-    
-    ABCD is the the hexadecimal representation of the data surrounded by < and >.
+    <ABCD>\n
```

Comment:
\n? :P

Reply:
The TCP packets used by the isotpserver include the line break..

Reply:
ha but we don't really take them into account, we could very well not have them